Created landing page and /map route

nataliet57 opened the pull request:
## Description
- Added npm library and react components to create a landing page 
- Added a /map route to access map simulation app (app is being rendered as a React Component via iFrame)
- Resizable window to accomadate for smaller screens

## To do: 
- Add a photo on landing page project on the landing page
- Remove optimal solution data
![image](https://github.com/thomaspaenns/EV-placement/assets/30819550/919d9ffb-3b5c-4ae9-af29-34ad0dad0403)
- Rename EV-placement -> voltpath or anything that doesn't have a dash because React doesn't like it very much 

## Screenshots
![image](https://github.com/thomaspaenns/EV-placement/assets/30819550/5da02b7c-7a52-4791-adb2-53113da30903)

![image](https://github.com/thomaspaenns/EV-placement/assets/30819550/0c3abd98-63ff-4ab5-bb89-aaf9d88a0807)

thomaspaenns reviewed (changes requested):
This looks awesome!  However, I'm not sure how to run it. Could you edit the readme and the install.sh file so that there are clear instructions for setting up and running the app?

One other thing, I think all these node modules should be included  in the gitignore file. 61k files in this PR seems like quite alot. I think it's better practice not to commit these ([link](https://dev.to/grawl/why-you-dont-have-to-commit-nodemodules-folder-33nm#:~:text=If%20you%20commit%20node_modules%20it,you%20have%20to%20solve%20that.))

nataliet57 commented:
> This looks awesome! However, I'm not sure how to run it. Could you edit the readme and the install.sh file so that there are clear instructions for setting up and running the app?
> 
> One other thing, I think all these node modules should be included in the gitignore file. 61k files in this PR seems like quite alot. I think it's better practice not to commit these ([link](https://dev.to/grawl/why-you-dont-have-to-commit-nodemodules-folder-33nm#:~:text=If%20you%20commit%20node_modules%20it,you%20have%20to%20solve%20that.))

Right! Totally agreed, added the node modules to the .gitignore and updated the README file. Take a look :)
